src/frontends/ir/src/os/lin/lin_mmap_object.cpp

```diff
@@ -64,7 +64,7 @@ class MapHolder {
     MapHolder() = default;
 
     void set(const std::string& path) {
-        int prot = PROT_READ;
+        int prot = PROT_READ | PROT_WRITE;
```

Comment:
will it affect performance? 
I suppose we don't want to write

Reply:
The reason it was added is because we already have transformations (`FlushFP32SubnormalsToZero`) which may change mapped memory. But mapped file is still in read-only mode, so we actually can't  write to disk

There should be no performance changes in default scenario because only when write there will be a copy of changed pages (not full .bin file will be copied)

Reply:
I additionally checked performance on small scope of models - no visible change for default read

Reply:
Maybe we need to change `FlushFP32SubnormalsToZero` not to perform inplace operation?

CC @ilyachur  what do you think? Looks like plugins change the original model instead of copying constant and work with copy.

Reply:
For the moment `FlushFP32SubnormalsToZero` is used only in MO, but we still need to support such possibility (`Copy-On-Write` mmap mode) because it will be less memory expensive than full copy of Constant data (and I guess slightly faster)

CC @slyalin

Reply:
I think we need to change the transformation. So nobody should change the model.

Reply:
As discussed, transformation will be updated, and `mmap()` will work in `page read only` mode